app/src/main/java/com/joelgh/features/rss_management/domain/RssRepository.kt

```diff
@@ -0,0 +1,6 @@
+package com.joelgh.features.rss_management.domain
+
+interface RssRepository {
+    fun create(rss: Rss)
+    fun delete(rss: Rss)
```

Comment:
Cuidadin que no se ha pedido que se elimine nada. ¿Qué pasa si se quiere sacar una versión sin que el usuario pueda eliminar el RSS? Esto es muy común ya que antes de implementar todas las funcionalidades prevista, se suele sacar una versión para conocer que aceptación tiene la funcionalidad entre los usuarios, recibir feedback y hacer cambios. 

Esto que has hecho ahora están común en el mundo del desarrollo que hasta tiene un nombre: YAGNI. Please, léete este link [Wikipedia - YAGNI](https://es.wikipedia.org/wiki/YAGNI) Este concepto está descrito en el libro Extreme Programming de Kent Beck.

Pongo en copia al compañero @albertoag128 que tambien es reponsable de haber añadido ``delete`` sin haberse pedido en el ticket.

Reply:
Asumo las consecuencias de haber dado por buena la pr. No estaba familiarizado con el concepto YAGNI (you are not going to need it), y tampoco vi ese metodo como un fallo o problema, aunque si que es cierto que deberíamos ceñirnos estrictamente a los tickets y a las funcionalidades que estos piden.

Reply:
He eliminado ya el método delete, después de leer el link que pasaste por aquí y el pdf que me compartiste por teams me ha quedado claro porque hay que hacer únicamente lo que se pide en el ticket